src/main/java/jag/Ui.java

```diff
@@ -0,0 +1,288 @@
+     *
+     * @param task Task instance that has been created, so that it's toString() method
+     *             can be called upon
+     * @param size Represents the updated size of the task list after the deletion
+     */
+    public void deleteResponse(Task task, int size) {
+        System.out.println(this.dashed);
+        System.out.println("Noted. I've removed this task:");
+        System.out.println(task.toString());
+        System.out.println("Now you have " + size + " tasks in the list.");
+        System.out.println(this.dashed);
+        command = scanner.nextLine();
+    }
+
+    /**
+     * Displays the right response to the user that a task has been unmarked
+     *
+     * @param task Task instance that has been created, so that it's toString() method
+     *             can be called upon
+     */
+    public void unmarkResponse(Task task) {
+        System.out.println(this.dashed);
+        System.out.println("OK, I've marked this task as not done yet:");
+        System.out.println(task.toString());
+        System.out.println(this.dashed);
+        command = scanner.nextLine();
+    }
+
+    /**
+     * Displays the right response to the user that a task has been marked
+     *
+     * @param task Task instance that has been created, so that it's toString() method
+     *             can be called upon
+     */
+    public void markResponse(Task task) {
+        System.out.println(this.dashed);
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println(task.toString());
+        System.out.println(this.dashed);
+        command = scanner.nextLine();
+    }
+
+
+    /**
+     * Display list of tasks that matches the description of the find
+     * command in the saved list, if size of the list is 0, that means
+     * there are no found tasks and a "sorry" message will be displayed
+     *
+     * @param foundTasks Represents an instance of a TaskList that
+     *                   consists of all the found tasks that was
+     *                   found by the FindCommand instance
+     */
+    public void findResponse(TaskList foundTasks) {
+        if (foundTasks.size() != 0) {
+            System.out.println(this.dashed);
+            System.out.println("Here are the matching tasks in your list: ");
+            for (int i = 0; i < foundTasks.size(); i++) {
+                System.out.println((i + 1) + ". " + foundTasks.getTask(i).toString());
+            }
+            System.out.println(this.dashed);
+            command = scanner.nextLine();
+        } else {
+            System.out.println(this.dashed);
+            System.out.println("Sorry we could not find any matching tasks in your list");
+            System.out.println(this.dashed);
+        }
+    }
+
+
+    /**
+     * Displays the response for exiting the chatbot
+     * and to close the instance of the scanner to end the application
+     */
+    public void exitResponse() {
```

Comment:
Another nit here, could this be renamed to something like displayExitResponse to more clearly resemble the use of a verb and explain more clearly?